test/utils.ts

```diff
@@ -49,20 +51,20 @@ export class Utils {
       outerHeight: 768,
       screenLeft: 0,
       screenTop: 0,
-      addEventListener: function (type: string, listener: (ev: MessageEvent) => void, useCapture?: boolean): void {
```

Comment:
Most of the changes here are formatting fixes from my IDE, but they seem to make the code cleaner. I could remove these if there's any issues though.